libass/ass_render.c

```diff
@@ -139,8 +139,9 @@ ASS_Renderer *ass_renderer_init(ASS_Library *library)
     priv->cache.bitmap_cache = ass_bitmap_cache_create();
     priv->cache.composite_cache = ass_composite_cache_create();
     priv->cache.outline_cache = ass_outline_cache_create();
+    priv->cache.face_size_metrics_cache = ass_face_size_metrics_cache_create();
     priv->cache.metrics_cache = ass_glyph_metrics_cache_create();
-    if (!priv->cache.font_cache || !priv->cache.bitmap_cache || !priv->cache.composite_cache || !priv->cache.outline_cache || !priv->cache.metrics_cache)
+    if (!priv->cache.font_cache || !priv->cache.bitmap_cache || !priv->cache.composite_cache || !priv->cache.outline_cache || !priv->cache.face_size_metrics_cache || !priv->cache.metrics_cache)
```

Comment:
This line is getting too long. Maybe it's better to move definition of `CacheStore` struct into `ass_cache.h` and coalesce all cache creation functions into one?

Reply:
This seems like a good idea, but I'm kinda inclined to defer it to a larger cache refactor post-threading (featuring e.g. a single shared LRU queue).

Reply:
Added breaks.